TheaterSchedule.DAL/Repositories/PerfomanceRepository.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.EntityFrameworkCore;
+using System.Linq;
+using System.Linq.Expressions;
+using Entities.Models;
+using TheaterSchedule.DAL.Interfaces;
+
+namespace TheaterSchedule.DAL.Repositories
+{
+    public class PerfomanceRepository: IPerfomanceRepository 
+    {
+        TheaterDatabaseContext db;
+        public PerfomanceRepository(TheaterDatabaseContext db)
+        {
+            this.db = db;
+        }
+       
+        public List<Performance> GetPerformanceTitles(int settingsId)

```

Comment:
this method needs only language
so pass only language instead of requiring full object with settings